examples/web_sys/todomvc/static/styles.css

```diff
@@ -0,0 +1,521 @@
+html,
```

Comment:
Could you move this into the showcase project dir?

Reply:
Ok yeah, since we aren't using it in the todo example